### PR TITLE
Make o2a installable in a fresh environment

### LINKED ISSUE #393: After installing from pypi in clean environment distcp_mapper is missing
**Describe the bug**
When you install o2a from pypi it misses distcp mapper when running

**To Reproduce**

1. Run pip install o2a
2. Run o2a --help
3. See the errorr (And accompanied info messages)

**Expected behavior**
It will run without problems.

**Additional context**
Seems that after pip install -e ., pip setup uses those earlier installed files locally to build the package. We need to make sure to clean it up and re-add all files if we want to prepare the whole package.

### PR DESCRIPTION
**Pull request by potiuk**
Fixes #393

Make sure you have checked _all_ steps below and that above description is correct.

### Github issue
- [x] My PR addresses the following [Issue](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/393)
issue and references it in commit message.

### Tests
- [x] My PR has unit tests testing the functionality (or I explained why it needs no tests)

### Commits
- [x] My commits reference the issue in the message using: Fixes #XXX /Closes #XXX or similar.

### Documentation
- [x] My PR adds documentation that describes how to use it (or I explained why it needs no docs)

### CODE REVIEW

File: o2a/converter/renderers.py [hunk 1]
```diff
@@ -144,6 +143,8 @@ def _remove_unused_imports(output_file_name):
 
     @staticmethod
     def _format_with_black(output_file_name):
+        import black
```
Comment (nuclearpinguin) [inline o2a/converter/renderers.py]: What is the reason behind this change?

Comment (potiuk) [reply o2a/converter/renderers.py]: Importing black at top level causes INFO messages being printed when you run o2a --help (bug in py2to3 library). I can separate it out if you think it's worth it :)

Comment (potiuk) [reply o2a/converter/renderers.py]: And BTW. The fix does not work well with tests :(

File: MANIFEST.in [hunk 2]
```diff
@@ -3,3 +3,11 @@ include bin/o2a-validate-workflows
 include o2a/schema/*.xsd
 include o2a/scripts/*.sh
 include o2a/templates/*.tpl
```
Comment (mik-laj) [inline MANIFEST.in]:
```suggestion
recursive-include o2a/templates *.tpl
```
WDYT?

Comment (potiuk) [reply MANIFEST.in]: Yep. I knew it must be possible :)

**Comment by potiuk** [comment]
We have two separate commits now and recursive-include :)
